course3/meeting2/hello.cc

```diff
@@ -1,13 +1,56 @@
 #include <iostream>
-#include <string>
 
+class SimpleString
+{
+  public:
+    SimpleString()
+    {
+    }
+
+    SimpleString(const char* ascii)
+    {
+      int lastCharacterIndex = BUFFER_SIZE - 1;
+
+      int i = 0;
+      while(i < lastCharacterIndex
```

Comment:
std::strcpy I can see, but why is for better than while in this case?

int i;
      for (i=0; i < lastCharacterIndex && ascii[i] != 0; ++i)
      {
        mBuffer[i] = ascii[i];
      }
      mBuffer[i] = 0;